src/OrbitGl/MainWindowInterface.h

```diff
@@ -32,6 +32,10 @@ class MainWindowInterface {
                                const std::string& assembly,
                                orbit_code_report::DisassemblyReport report) = 0;
 
+  enum class CaptureLogSeverity { kInfo, kWarning, kSevereWarning, kError };
+  virtual void AppendToCaptureLog(CaptureLogSeverity severity, std::string_view capture_time,
```

Comment:
From an interface perspect I would expect the capture_time to be a chrono or abseil timestamp or duration, not a string, such that formatting happens in a single place. Not sure though if that change is worth it. I assume you built this around existing functions which return strings?

Reply:
Yep, basically you said it all, I agree the interface is weird, but at the same time this uses `OrbitApp`'s `GetCaptureTime` and the new `GetCaptureTimeAt`, which already return formatted strings. 

Internally these methods call `GetPrettyTime` so I could just call that directly in `OrbitMainWindow` (although `GetPrettyTime` is currently only called in `OrbitGl`). That would also allow the capture log to choose to change the way it formats timestamps. In that case I think that `GetCaptureTime` should just be changed to return an `absl::Duration` and all the places it's called from should use `GetPrettyTime` (there are not that many).

What do you think?

Reply:
Yes, that sounds perfect to me!

Reply:
Done.